yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -60,13 +60,16 @@ func prepareForImportDataCmd() {
 		ExportDir:     exportDir,
 	}
 	if fileOptsMap["quote_char"] != "" {
-		dfd.QuoteChar = byte(fileOptsMap["quote_char"][0])
+		quoteCharBytes := []byte(fileOptsMap["quote_char"])
+		dfd.QuoteChar = quoteCharBytes[0]
 	}
 	if fileOptsMap["escape_char"] != "" {
-		dfd.EscapeChar = byte(fileOptsMap["escape_char"][0])
+		escapeCharBytes := []byte(fileOptsMap["escape_char"])
+		dfd.EscapeChar = escapeCharBytes[0]
 	}
 	dfd.Save()
 
+	escapeFileOptsCharsIfRequired() //escaping for COPY command should be done after saving chars in descriptor
```

Comment:
nit:
`//escaping for COPY command should be done after saving chars in descriptor`
->
`// escaping for COPY command should be done after saving fileOpts in data file descriptor`